subsys/bindesc/Kconfig.version

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023 Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BINDESC_DEFINE_VERSION
+	bool "Version binary descriptors"
+	help
+	  Add the version binary descriptors
+
+if BINDESC_DEFINE_VERSION
+
+config BINDESC_KERNEL_VERSION_STRING
```

Comment:
This is already generated from the zephyr VERSION file, it should be used instead instead of duplicating here. For an *application* version there is an RFC for adding it https://github.com/zephyrproject-rtos/zephyr/issues/56605 with 2 possible implementations, likewise this should use that rather than duplicating it

Reply:
This is not the actual version, this is a boolean that selects whether you want it as a descriptor or not